include/aos/common/cloudprotocol/state.hpp

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (C) 2025 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CLOUDPROTOCOL_STATE_HPP_
+#define AOS_CLOUDPROTOCOL_STATE_HPP_
+
+#include "aos/common/crypto/crypto.hpp"
+#include "aos/common/tools/optional.hpp"
+#include "aos/common/types.hpp"
+
+namespace aos::cloudprotocol {
+
+/**
+ * State length.
+ */
+static constexpr auto cStateLen = AOS_CONFIG_CLOUDPROTOCOL_STATE_LEN;
+
+/**
+ * State reason length.
+ */
+static constexpr auto cStateReason = cErrorMessageLen;
+
+/**
```

Comment:
It could be one enum for all cloud messages I guess. Do you need this enum right now?